<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NTUCircleDetectWidget</class>
 <widget class="QGroupBox" name="NTUCircleDetectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Tool</string>
  </property>
  <property name="title">
   <string>Edit Tool</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tool Enable:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tool Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_toolEnable"/>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_toolFixture">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tool Fixture:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Execution Time (ms):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_executionTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>480</width>
            <height>213</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="7" column="0">
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Result:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Edge Transition:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Edge Contrast:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="3">
              <widget class="QSpinBox" name="spinBox_edgeWidth">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="value">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Out Region:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="3">
              <widget class="QSpinBox" name="spinBox_edgeContrast">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="6" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEdit_in">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="7" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEdit_result">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="3">
              <widget class="QSpinBox" name="spinBox_timeOut">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
               <property name="value">
                <number>5000</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>In Region:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="QComboBox" name="comboBox_edgeTransition">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>Black to White</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>White to Black</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Both</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Edge Width:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>TimeOut (ms):</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEdit_out">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Threshold Method:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="3">
              <widget class="QComboBox" name="comboBox_thresholdMethod">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>Manual</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Auto</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
